hazelcast-sql/pom.xml

```diff
@@ -831,7 +831,7 @@
         <dependency>
             <groupId>com.microsoft.sqlserver</groupId>
             <artifactId>mssql-jdbc</artifactId>
-            <version>12.4.0.jre8-preview</version>
+            <version>${mssql.version}</version>
```

Comment:
The same change needs to be done in the mapstore module.

Also, you can omit the version because it is defined in the dependency management section, but other dependencies don't do that, so let's keep as is.